src/components/Account/Buttons/ChangeNetworkModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import {
+  GU,
+  Modal,
+  useViewport,
+  IconConnect,
+  textStyle,
+  ButtonBase,
+} from '@1hive/1hive-ui'
+import { switchNetwork } from '@/networks'
+
+type ChangeNetworkModalProps = {
+  compact: any
+  visible: boolean
+  onClose: () => void
+}
+
+export default function ChangeNetworkModal({
+  compact,
+  visible,
+  onClose,
+}: ChangeNetworkModalProps) {
+  const { width } = useViewport()
+
+  const handleSwitch = async () => {
+    try {
+      await switchNetwork(100)
+      onClose()
```

Comment:
if you `throw new Error('some error test')` it show some popscreen error? the default one with sad flower? Just curious

Reply:
I don't really know. But I will check it.

Reply:
Basically the throw new Error() appears like this:

<img width="1208" alt="Captura de ecrã 2022-02-26, às 17 30 48" src="https://user-images.githubusercontent.com/13184907/155853137-4957d83a-2f25-4ba2-b381-e09de3734b89.png">

Reply:
So nothing in the screen? We need handle that error, what you think?

Reply:
Just like this. 

<img width="678" alt="Captura de ecrã 2022-02-26, às 18 40 29" src="https://user-images.githubusercontent.com/13184907/155855212-221dd750-b76c-42f3-93ef-d10cc6103266.png">

Reply:
Awesome! My bad not follow the code inside the method.